lib/pavilion/filters.py

```diff
@@ -224,13 +228,55 @@ def add_series_filter_args(arg_parser: argparse.ArgumentParser,
                 )
 
     if sort_functions is None:
-        sort_functions = SERIES_SORT_FUNCS.copy()
+        sort_functions = SORT_KEYS["SERIES"]
 
-    sort_options = (list(sort_functions.keys())
-                    + ['-' + key for key in sort_functions.keys()])
+    sort_options = (sort_functions
+                    + ['-' + key for key in sort_functions])
 
     add_common_filter_args("series", arg_parser, defaults, sort_options)
 
+#  select once so we only make one filter.
+def filter_test_run(test_attrs, cmp, fail, incmp, nm, nt, ot, pss, rslte, ss, sn, usr):
```

Comment:
While I understand the desire for brevity, these arguments need to be longer and more readable, and documented in the docstring.

Reply:
Got it.  Seems like it works now I had some issues with the docstrings.